### PR TITLE
batman-adv-legacy: use recent batctl instead of batctl v2013.4

### PR DESCRIPTION
**Pull request by T-X**
With the netlink functions backported batman-adv-legacy works fine with the upstream batctl. Therefore there is no more need to build and install the old batctl v2013.4 version.

### CODE REVIEW

**Comment by T-X** [comment]
Depends on: https://github.com/freifunk-gluon/packages/pull/213

This, together with https://github.com/freifunk-gluon/batman-adv-legacy/pull/15 was tested on a node with a Freifunk Lübeck site configuration, which uses batman-adv-legacy. The network comes up fine and meshes with other nodes, the "batctl o" output looks sane and "batctl dc -H" and "batctl tl -H" as used by gluon-ebtables-limit-arp seems to work correctly.

File: package/gluon-mesh-batman-adv/Makefile [hunk 1]
```diff
@@ -7,7 +7,7 @@ include ../gluon.mk
 
 define Package/gluon-mesh-batman-adv/common
   PROVIDES:=gluon-mesh-batman-adv
-  DEPENDS:=+gluon-core +libgluonutil +gluon-client-bridge +gluon-ebtables +firewall +libiwinfo +kmod-dummy +libnl-tiny +libbatadv +@GLUON_SPECIALIZE_KERNEL:KERNEL_DUMMY +@GLUON_SPECIALIZE_KERNEL:KERNEL_CRC16 +@GLUON_SPECIALIZE_KERNEL:KERNEL_LIBCRC32C
+  DEPENDS:=+gluon-core +libgluonutil +gluon-client-bridge +gluon-ebtables +firewall +libiwinfo +kmod-dummy +libnl-tiny +libbatadv +batctl +@GLUON_SPECIALIZE_KERNEL:KERNEL_DUMMY +@GLUON_SPECIALIZE_KERNEL:KERNEL_CRC16 +@GLUON_SPECIALIZE_KERNEL:KERNEL_LIBCRC32C
```
Comment (NeoRaider) [inline package/gluon-mesh-batman-adv/Makefile]: While we're at it, split this line into multiple?

**Comment by T-X** [comment]
Changelog v2:
* updated modules hash to new changes in freifunk-gluon/packages#213
* used newlines for dependencies

**Comment by T-X** [comment]
Changelog v3:
* updated modules hash to new changes in freifunk-gluon/packages#213

**Comment by T-X** [comment]
Changelog v4:
* updated modules hash to new changes in freifunk-gluon/packages#213

**Comment by rotanid** [comment]
sorry, the way i merged your other PR doesn't work with your previous update to this PR, it has to be updated again.

**Comment by T-X** [comment]
Changelog v5:
* updated "modules: update Gluon packages" to current freifunk-gluon/packages
* rebase to current Gluon master

**Comment by rotanid** [comment]
done

**Pull request merged**